test/physics/em/ImportedProcesses.test.cc

```diff
@@ -196,6 +197,40 @@ TEST_F(ImportedProcessesTest, photoelectric)
     }
 }
 
+TEST_F(ImportedProcessesTest, bremsstrahlung)
+{
+    // Create photoelectric process (requires Geant4 environment variables)
+    std::shared_ptr<BremsstrahlungProcess> process;
+    try
+    {
+        process = std::make_shared<BremsstrahlungProcess>(
+            particles_, materials_, processes_);
+    }
+    catch (const RuntimeError& e)
+    {
+        SKIP("Failed to create process: " << e.what());
+    }
+
+    // Test model
+    auto models = process->build_models(ModelIdGenerator{});
+    ASSERT_EQ(1, models.size());
+    ASSERT_TRUE(models.front());
+    EXPECT_EQ("Seltzer-Berger bremsstrahlung", models.front()->label());
+    auto all_applic = models.front()->applicability();
+    ASSERT_EQ(1, all_applic.size());
+    Applicability applic = *all_applic.begin();
+
+    // Test step limits
+    for (auto mat_id : range(MaterialId{materials_->num_materials()}))
+    {
+        applic.material = mat_id;
+        auto builders   = process->step_limits(applic);
+        EXPECT_TRUE(builders[VGT::macro_xs]);
+        EXPECT_FALSE(builders[VGT::energy_loss]);
```

Comment:
I thought brems has a de/dx component for electrons below the production threshold?

Reply:
This is true.